src/get-started/java-binding.md

```diff
@@ -522,39 +520,30 @@ For using the library just include the dependency in your `pom.xml`:
-<!--TODO: Insert the correct Javadoc (once we publish things): -->
-[transaction-execution-context]: https://exonum.com/doc/api/java-binding-core/latest/com/exonum/binding/transaction/Transaction.html
+[how-to-build]: https://github.com/exonum/exonum-java/blob/v0.4/CONTRIBUTING.md#how-to-build
+[Memorydb]: https://exonum.com/doc/api/java-binding-core/0.4/com/exonum/binding/storage/database/MemoryDb.html
+[nodefake]: https://exonum.com/doc/api/java-binding-core/0.4/com/exonum/binding/service/NodeFake.html
+[schema]: https://exonum.com/doc/api/java-binding-core/0.4/com/exonum/binding/service/Schema.html
+[service]: https://exonum.com/doc/api/java-binding-core/0.4/com/exonum/binding/service/Service.html
+[service-after-commit]: https://exonum.com/doc/api/java-binding-core/0.4/com/exonum/binding/service/Service.html#afterCommit(com.exonum.binding.service.BlockCommittedEvent)
+[node-submit-transaction]: https://exonum.com/doc/api/java-binding-core/0.4/com/exonum/binding/service/Node.html#submitTransaction(com.exonum.binding.transaction.RawTransaction)
+[standardserializers]: https://exonum.com/doc/api/java-binding-common/0.4/com/exonum/binding/common/serialization/StandardSerializers.html
+[storage-indices]: https://exonum.com/doc/api/java-binding-core/0.4/com/exonum/binding/storage/indices/package-summary.html
+[transaction]: https://exonum.com/doc/api/java-binding-core/0.4/com/exonum/binding/transaction/Transaction.html
+[transaction-execution-context]: https://exonum.com/doc/api/java-binding-core/0.4/com/exonum/binding/transaction/TransactionContext.html
 [transactions]: ../architecture/transactions.md
-[transactionconvererter]: https://exonum.com/doc/api/java-binding-core/latest/com/exonum/binding/service/TransactionConverter.html
+[transactionconvererter]: https://exonum.com/doc/api/java-binding-core/0.4/com/exonum/binding/service/TransactionConverter.html
 [vertx.io]: https://vertx.io/docs/vertx-web/java/#_basic_vert_x_web_concepts
 [vertx-web-client]: https://vertx.io/docs/vertx-web-client/java
 [maven-install]: https://maven.apache.org/install.html
-[cryptofunctions-ed25519]: https://exonum.com/doc/api/java-binding-common/latest/com/exonum/binding/common/crypto/CryptoFunctions.html#ed25519--
-[createpublicapi]: https://exonum.com/doc/api/java-binding-core/latest/com/exonum/binding/service/Service.html#createPublicApiHandlers-com.exonum.binding.service.Node-io.vertx.ext.web.Router-
-[transaction-result]: https://docs.rs/exonum/latest/exonum/blockchain/type.TransactionResult.html
+[cryptofunctions-ed25519]: https://exonum.com/doc/api/java-binding-common/0.4/com/exonum/binding/common/crypto/CryptoFunctions.html#ed25519--
+[createpublicapi]: https://exonum.com/doc/api/java-binding-core/0.4/com/exonum/binding/service/Service.html#createPublicApiHandlers-com.exonum.binding.service.Node-io.vertx.ext.web.Router-
+[transaction-result]: https://docs.rs/exonum/0.4/exonum/blockchain/type.TransactionResult.html
```

Comment:
Not ours.

Reply:
Thanks. Btw it would be indeed a good idea to check links automatically, as https://docs.rs/exonum/latest/exonum/blockchain/type.TransactionResult.html is not valid anymore, it's https://docs.rs/exonum/latest/exonum/blockchain/struct.TransactionResult.html

Reply:
> Btw it would be indeed a good idea to check links automatically

exonum-doc supposedly does that, doesn't it?

Anyway, using versioned links seems to be more robust, but we need to update them each release.

Reply:
```
That's it. 4108 links in 52 URLs checked. 0 warnings found. 0 errors found.
Stopped checking at 2019-02-13 23:59:07+000 (17 seconds)
```

I don't know what it checks, but there is no v0.4 tag in our repository at the moment :smiling_imp: